MinPlatformPkg/Test/Library/TestPointCheckLibNull/TestPointCheckLibNull.c

```diff
@@ -657,6 +677,29 @@ TestPointSmmReadyToLockSmmMemoryAttributeTableFunctional (
   return EFI_SUCCESS;
 }
 
+/**
+  This service verifies the validity of the StandaloneMm memory atttribute table at StandaloneMm 
+  Ready To Lock.
```

Comment:
Typo in `atttribute`.

```suggestion
  This service verifies the validity of the StandaloneMm memory attribute table at StandaloneMm 
  Ready To Lock.
```

Reply:
Fixed.